select2-parent/select2/src/main/java/org/wicketstuff/select2/AbstractSelect2Choice.java

```diff
@@ -273,7 +275,14 @@ protected void onInitialize()
 	protected void onConfigure()
 	{
 		super.onConfigure();
-		if (isAjax())
+        if(getSettings().isStateless()) 
```

Comment:
seems to be mixture of space/tab delimiters
all code has spaces, so I would leave it consistent